src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -108,16 +108,15 @@
                                Text="{Binding Dimensions}"
                                Grid.Row="0"
                                Margin="0,0,0,0"
-                               FontSize="11"
-                               FontWeight="SemiBold"
+                               FontSize="14"
                                HorizontalAlignment="Center"
                                VerticalAlignment="Center"
                                Foreground="{Binding (TextElement.Foreground), RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type ContentPresenter}}}" />
                     <TextBlock Name="ScalingText"
                                TextTrimming="CharacterEllipsis"
                                Text="{Binding Scaling}"
                                Margin="0,0,0,0"
-                               FontSize="8"
+                               FontSize="12"
```

Comment:
![image](https://user-images.githubusercontent.com/9866362/230791245-1836d25b-0a26-4443-9bfc-f39f48487a95.png)

Reply:
I'll have to search for the cause for this. I suppose you don't have a custom setting for UI text size or something? Looked fine when I was building on Win 10 and running on Win 11
![image](https://user-images.githubusercontent.com/65828559/230795288-ddaf7441-e5f2-406b-a7b9-3b5f4e68120d.png)

Reply:
I am trying this, but it fails when running and I don't know why.
```xaml
                <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
                    <Grid.RowDefinitions>
                        <RowDefinition Height="*"/>
                        <RowDefinition Height="Auto"/>
                    </Grid.RowDefinitions>
                    <TextBlock Name="IndexText"
                        Grid.Row="0"
                        Text="{Binding Index}"
                        TextTrimming="CharacterEllipsis"
                        FontSize="28"
                        FontWeight="SemiBold"
                        HorizontalAlignment="Center"
                        VerticalAlignment="Center"
                        Foreground="{Binding (TextElement.Foreground), ElementName=MonitorItem}" />

                    <TextBlock
                        Grid.Row="1"
                        HorizontalAlignment="Center"
                        TextTrimming="CharacterEllipsis"
                        FontSize="12"
                        >
                        <Run Text="{Binding Dimensions}"/>
                        <Run>@</Run>
                        <Run Text="{Binding Scaling}"/>
                    </TextBlock>
                </Grid>
```
@SeraphimaZykova ?

Reply:
Those rectangles keep monitor aspect ratio to help differentiate them. You can change `MinPreviewDisplaySize` to make space for a larger font.
https://github.com/microsoft/PowerToys/blob/893719859dfbec53dd88db9077b20080e2c86eaa/src/modules/fancyzones/editor/FancyZonesEditor/ViewModels/MonitorViewModel.cs#L14

Reply:
Thanks @SeraphimaZykova I changed it to 120 now. Need @niels9001 to confirm on his computers. I still have no idea why it looked the way it did, for you. I can't reproduce it. Note that this text does not respond to Windows Acessibility settings > Text size.
![image](https://user-images.githubusercontent.com/65828559/231488293-f7bc0360-7bc0-4a72-be24-7f9ae8a6b73c.png)

What I meant by the previous comment is that the `<Run Name="ResolutionText" Text="{Binding Dimensions}"/>` will crash when opening the Editor. Any idea why that could happen?

Reply:
Also, for testing with different dimensions, you can manually set "monitor-width" and "monitor-height" in the `editor-parameters.json` before running editor from Visual Studio. So you'll see how it will look on 3440x1440 or any other monitor. It's important to run editor alone to test this way, because FZ will rewrite the file when you press a shortcut.

Reply:
Thank you, that helps alot! :) Now I see it's the _MaxPreview_ that causes the issue. "Relative scaling" between the two.
![image](https://user-images.githubusercontent.com/65828559/231505941-1ab16708-4a9c-4a15-9ec7-8228e6a295ed.png)